### PR TITLE
FEAT: OmniSciDB Log Log2 Log10 implementations

### PR DESCRIPTION
**Pull request by semelianova**

### CODE REVIEW

**Comment by jreback** [comment]
these PRa are good
can i add a release note (you can add onto the same one if applicable)

File: ci/docker-compose.yml [hunk 1]
```diff
@@ -89,7 +89,7 @@ services:
       KUDU_MASTER: "false"
 
   omniscidb:
-    image: omnisci/core-os-cpu:v5.1.0
+    image: omnisci/core-os-cpu-dev:latest
```
Comment (jreback) [inline ci/docker-compose.yml]: @xmnlab is this right? why are we not pinning exactly (to 5.2?)

File: ci/requirements-3.6-dev.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ dependencies:
   - black=19.10b0
   - click
   - clickhouse-cityhash
-  - clickhouse-driver>=0.0.8,<0.1.2
+  - clickhouse-driver>=0.1.3
```
Comment (jreback) [inline ci/requirements-3.6-dev.yml]: these are from another PR (I get that you changed them to make things pass, but pls revert)

File: ibis/omniscidb/operations.py [hunk 3]
```diff
@@ -289,6 +290,37 @@ def extract_field_formatter(translator, expr):
     return extract_field_formatter
 
 
+def _log_common(translator, arg, base=None):
```
Comment (jreback) [inline ibis/omniscidb/operations.py]: can use add a comment like ``# NUMERIC`` here before this section (are there other ops where this should go, e.g. sin, cos, IIRC we have a PR about this, but not merged)

File: setup.py [hunk 4]
```diff
@@ -26,7 +26,7 @@
 kerberos_requires = ['requests-kerberos']
 visualization_requires = ['graphviz']
 clickhouse_requires = [
-    'clickhouse-driver>=0.0.8,<0.1.2',
+    'clickhouse-driver>=0.1.3',
```
Comment (jreback) [inline setup.py]: same here

File: ci/docker-compose.yml (hunk 1 above)
Comment (semelianova) [reply ci/docker-compose.yml]: My several pull requests aren't working with the latest omnisci release due to a lack of some changes.

**Comment by jreback** [comment]
pls merge master

File: ci/docker-compose.yml (hunk 1 above)
Comment (xmnlab) [reply ci/docker-compose.yml]: this was discussed on https://github.com/ibis-project/ibis/pull/2084

**Comment by jreback** [comment]
can you merge master, update the release.rst, and remove the change in the image.

**Comment by semelianova** [comment]
Need to wait for #2084 is merged

File: ibis/omniscidb/operations.py [hunk 5]
```diff
@@ -1037,6 +1075,7 @@ def formatter(translator, expr):
     ops.Greatest,
     ops.Log2,
     ops.Log,
+    ops.Round,
```
Comment (jreback) [inline ibis/omniscidb/operations.py]: was this just missing before?

Comment (semelianova) [reply ibis/omniscidb/operations.py]: yes

**Comment by jreback** [comment]
thanks @semelianova

**Pull request merged**